docs/types.rst

```diff
@@ -413,8 +413,18 @@ function type should not return anything, the whole ``returns (<return types>)``
 part has to be omitted.
 
 By default, function types are internal, so the ``internal`` keyword can be
-omitted. In contrast, contract functions themselves are public by default,
-only when used as the name of a type, the default is internal.
+omitted. In contrast, visibility has to be specified for contract functions themselves,
+only when used as a type, there is a default of internal.
```

Comment:
Maybe a full stop before this line would make it easier to read? Also why ``themselves``?
```
In contrast, visibility has to be specified for all contract functions.
For function types the visibility defaults to internal.
```
By the way: should we actually remove the exception for function types here? Might be more consistent.